### PR TITLE
barebones track generator

### PR DESCRIPTION
**Pull request by nerik**
A very simple track Generator.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "@mapbox/tilebelt": "^1.0.1",
     "@mapbox/vector-tile": "^1.3.1",
+    "@types/geojson": "^7946.0.7",
```
Comment (j8seangel) [inline package.json:49]: wow, this is a huge version

File: src/layer-composer/generators/track/track.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { GeneratorConfig } from 'layer-composer/types'
+// TODO custom "augemented" GeoJSON type
+// see https://github.com/yagajs/generic-geojson/blob/master/index.d.ts
```
Comment (j8seangel) [inline src/layer-composer/generators/track/track.ts:3]: 👍

**Review by j8seangel — APPROVED** [review]

File: src/layer-composer/types.ts [hunk 3]
```diff
@@ -33,3 +33,8 @@ export interface GeneratorConfig {
   visible?: boolean
   opacity?: number
 }
+
+export interface GlobalGeneratorConfig {
+  start?: string
```
Comment (j8seangel) [inline src/layer-composer/types.ts:40]: I've been doing some research to ensure this is an isoString but typescript doesn't allow it... only checks types... 
so what about using something like this https://github.com/typestack/class-validator to ensure this uses the desired format (https://github.com/typestack/class-validator/blob/90a663866eaeb5a0cb38cce5501e0aa28984ed34/src/validation/Validator.ts#L401)

**Review by j8seangel — APPROVED** [review]
Looks good, having this global config should simplify each generator.

File: src/layer-composer/types.ts (hunk 3 above)
Comment (nerik) [reply src/layer-composer/types.ts:40]: I like the idea but I'm not sure how this can apply to TS interfaces?

**Comment by j8seangel** [comment]
Merging to release this but pending to talk about the class fields validations

**Pull request merged**